src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__label {
+  text-align: right;
+  padding-left: 17px;
+  font-size: 10px;
+  color: $color_text_main;
+  line-height: 18px;
+}
+
+
+.card__stars {
+  display: flex;
+
+  &-star{
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+  &:not(:last-child) {
+    background-image: url('../images/star-active.svg')
+  }
```

Comment:
it's better to keep modificators for all possible active stars (as in the previous task)  - FYI (optionally)